Tasks/DotNetCoreCLIV2/restorecommand.ts

```diff
@@ -135,8 +138,7 @@ export async function run(): Promise<void> {
             nuGetConfigHelper.restoreBackupRootNuGetFiles();
         }
 
-        tl.setResult(tl.TaskResult.Succeeded, tl.loc('PackagesInstalledSuccessfully'));
-
+        setTaskResultOnNugetBehavior(includeNuGetOrg);
```

Comment:
We shouldn't wait until this point to fail. A more tactical change here would be to get and respond to the behavior flag right at the point of usage, i.e. line 108 where it actually adds nuget.org as a source. If the goal is to warn, can we output a warning message at that point and then use some "hasWarnings" flag to determine the final result should be SucceededWithIssues? If the goal is to fail, we should fail immediately at that point so that no restore happens.

Reply:
fixed! lmk if you like